spec/integrations/posts/index_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index', type: :system do
+  before(:each) do
+    @user = User.create(name: 'Elham',
+                        photo: 'https://media.istockphoto.com/id/1406197730/photo/portrait-of-a-young-handsome-indian-man.webp?b=1&s=170667a&w=0&k=20&c=KtmKHyOE6MJV0w2DiGX8P4399KHNbZ3p8lCjTEabGcY=',
+                        bio: 'Best friends', posts_counter: 0)
+    @post1 = @user.posts.create(title: 'Post 1', text: 'This is the first post.', comments_counter: 0, likes_counter: 0)
+    @post2 = @user.posts.create(title: 'Post 2', text: 'This is the second post.', comments_counter: 0,
+                                likes_counter: 0)
+    @post3 = @user.posts.create(title: 'Post 3', text: 'This is the third post.', comments_counter: 0, likes_counter: 0)
+    Comment.create(post: @post1, author: @user, text: 'This is a comment')
+    Like.create(post: @post1, author: @user)
+  end
+  describe 'post page' do
+    before(:each) do
+      visit(user_posts_path(@user.id))
+    end
+    it "shows user's profile picture" do
+      expect(page).to have_css('img[src*="https://media.istockphoto.com/id/1406197730/photo/portrait-of-a-young-handsome-indian-man.webp?b=1&s=170667a&w=0&k=20&c=KtmKHyOE6MJV0w2DiGX8P4399KHNbZ3p8lCjTEabGcY="]')
+    end
+
```

Comment:
- [ ] As required for the project, kindly ensure that all your tests are passing, as  right now the selected one above is not passing.
![Screenshot from 2023-08-03 15-38-29](https://github.com/Elhambasir/blog-app/assets/83042742/7431c8b7-9661-4e71-80cb-7128d30bb76a)

Reply:
Thank you I will fix it

Reply:
The test for `integrations/posts/index_spec.rb` is passing on my side you can check the [video](https://www.loom.com/share/16e6893b2ef34b78ad60ab08df8450ac?sid=d09cdc9d-ba9e-4757-99e1-7698ab0a3951).